_docs/howto/configuring-forseti.md

```diff
@@ -46,6 +46,81 @@ If you change a configuration, such as by changing which pipeline or scanner to
 run, you'll need to restart the appropriate module so it picks up the new
 configuration change.
 
-## What's next
 
--   Read about [configuring Inventory]({% link _docs/quickstarts/inventory/#configuring-inventory %}).
+### Configuring Inventory
+
+{% include _howto/config_inventory.md %}
+
+#### Setting up batch mode
+
+You can run Forseti Inventory in batch mode to execute multiple inventory
+pipelines to in the same run. To specify which pipelines to run:
+
+1.  Open `forseti-security/configs/forseti_confs.yaml`.
+1.  Navigate to the `inventory` section.
+1.  Edit the `enabled` flag for the appropriate pipelines. Set the flag to
+    `true` to enable a pipeline, or `false` to disable the pipeline.
+
+````
+inventory:
+
```

Comment:
I would consider leaving this section out because as we add new pipelines, it will get outdated and we have to keep updating it.